changelog/Microsoft.IdentityProtectionServices.json

```
@@ -1,5 +1,23 @@
 {
 	"changelog": [
+		{
+      			"ChangeList": [
+        			{
+					  "Id": "8e875dc0-5df2-43c8-8c6b-316aed281493",
+					  "ApiChange": "Member",
+					  "ChangedApiName": "linux",
+					  "ChangeType": "Addition",
+					  "Description": "Added the **linux** member to the **conditionalAccessDevicePlatform** enumeration.",
```

Suggestion to enhance the description and give the customer added context
```suggestion
					  "Description": "Added the **linux** member to the **conditionalAccessDevicePlatform** enumeration. This update allows devices running the Linux platform to be targeted for inclusion or exclusion in conditional access policies.",
```